app/lib/common/widgets/spaces/space_with_profile_card.dart

```diff
@@ -107,14 +106,16 @@ class SpaceWithProfileCard extends StatelessWidget {
           displayName: title,
           avatar: profile.getAvatarImage(),
         ),
-        parentBadges: showParent && parentRoomId != null
-            ? [
-                AvatarInfo(
-                  uniqueId: parentRoomId!,
-                  displayName: parentProfile?.displayName ?? parentRoomId,
-                  avatar: parentProfile?.getAvatarImage(),
-                ),
-              ]
+        parentBadges: showParents && (parents != null && parents!.isEmpty)
```

Comment:
```suggestion
        parentBadges: showParents && parents?.isEmpty != false
```

Reply:
This'd lead to null error. This validates it... 
```
showParents && (parents?.isEmpty ?? false)
```